topics/mongoose/index.md

```diff
@@ -0,0 +1,11 @@
+---
+created_by: LearnBoost
+display_name: Mongoose
+github_url: https://github.com/Automattic/mongoose
+logo: mongoose.png
+released: 2010
```

Comment:
You could add a `related: mongodb, nodejs, orm` line.

Reply:
added